docs/content/new-tutorial/intro-to-assets.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: "Introduction to the Dagster Tutorial and Assets | Dagster Docs"
+description: "Learn about the objectives for the Dagster tutorial and about assets"
+noindex: True
+---
+
+# Introduction to Dagster
+
+Dagster is an orchestration framework to help data engineers deliver high-quality data. In Dagster, the objects made by data engineers are called assets.
+
+In this tutorial, you will write a data pipeline that performs ETL (Extract, Transform, Load). You will:
+
+- **Extract** data from a website
+- **Transform** it into a data structure for a chart
+- And **load** it into a visualization for reporting
+
+While writing this pipeline, you will learn about Dagster's core concepts, such as assets, schedules, and I/O Managers.
+
+## Introduction to software-defined assets
+
+Before building a Dagster project, you should first learn about Dagster's primary core concept: the Software-defined asset (SDA).
+
+Assets are objects saved somewhere and are the outputs of data pipelines. Assets can be any type of object, such as:
+
+- A database table or view
+- A file, such as in your local machine or blob storage like S3
+- A machine learning model
+- An object in an integration (such as a dbt model or an Airbyte-ingested table)
+
+A Software-defined asset is Dagster's concept that couples an asset to the function and upstream assets that are used to produce its contents.
+
+<Image
+alt="A materialized asset shown through the Dagit UI"
+src="/images/tutorial/simple-asset.png"
+width={275}
+height={132}
+/>
+
+If you have an existing data pipeline, you likely already have assets.
+
+### Goals
+
+By the end of this section, you should:
+
+- Know what assets are
+- Know the power of Software-defined assets
+- Know when to use task-based orchestration with assets
+- Begin to think about what parts of your current pipeline are assets.
+
+### Parts of a software-defined asset
+
+Software-defined assets are powerful because they encapsulate objects to enable data engineers to do more with them.
+
+As a Dagster user, you define assets in code. Developers can set multiple properties to add more information to the asset. For example, software-defined assets enrich assets by being able to have:
+
+- A description that end users will understand what the asset is for
+- The lineage of what other assets it relates to
+- A group name to categorize multiple assets or a Compute Kind to group assets by the origins of the data, such as an ingestion tool.
```

Comment:
These are advanced concepts that I think we can omit from the tutorial.

Reply:
I think I feel strongly that they shouldn't be on this page and weakly that they shouldn't be in the tutorial at all. Very possible that they could end up fitting in later in the tutorial.  Alternatively, might make sense to have the end of the tutorial link to a guide on "Organizing Assets" or something.